projects/ngrx.io/content/guide/signals/rxjs-integration.md

```diff
@@ -1,3 +1,98 @@
-# RxJS Integration
+# Opt-In Interopability with RxJS
 
-**UNDER CONSTRUCTION**
+RxJS is still a major part of NgRx and the Angular ecosystem, and the NgRx Signals package provides **opt-in** usage to interact with RxJS observables using the rxMethod function.
+
+The `rxMethod` function allows you to define a method that can receive a signal or observable, read its latest values, and perform additional operations with an observable.
+
+```ts
+import { inject } from '@angular/core';
+import { debounceTime, distinctUntilChanged, pipe, switchMap, tap } from 'rxjs';
+import {
+  signalStore,
+  patchState,
+  withHooks,
+  withMethods,
+  withState,
+} from '@ngrx/signals';
+import { rxMethod } from '@ngrx/signals/rxjs-interop';
+import { tapResponse } from '@ngrx/operators';
+
+import { User } from './user.model';
+import { UsersService } from './users.service';
+
+type State = { users: User[]; isLoading: boolean; query: string };
+
+const initialState: State = {
+  users: [],
+  isLoading: false,
+  query: '',
+};
+
+export const UsersStore = signalStore(
+  { providedIn: 'root' },
+  withState(initialState),
+  withMethods((store, usersService = inject(UsersService)) =&gt; ({
+    updateQuery(query: string) {
+      patchState(store, { query });
+    },
+    async loadAll() {
+      patchState(store, { isLoading: true });
+      const users = await usersService.getAll();
+      patchState(store, { users, isLoading: false });
+    },
```

Comment:
This method is not relevant to RxJS interop/integration, so I think we can remove it to have the focused example:

```suggestion
```